config/prow/cluster/build/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -146,9 +146,6 @@ spec:
   backendType: gcpSecretsManager
   projectId: cloud-provider-vsphere
   data:
-  - key: GCR_KEY_FILE
```

Comment:
/hold 

We have to keep this for a bit until CSI stops using the corresponding preset

Once that is done we can probably delete even more